dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/RowOrganizedStachExtension.cs

```diff
@@ -136,8 +138,26 @@ private Models.Table GenerateTable(RowOrganizedPackage.Types.Table table)
 
             foreach (var metadataItem in table.Data.TableMetadata)
             {
-                var metadataValue = StachUtilities.ValueToString(metadataItem.Value.Value);
-                finalTable.Metadata.Add(metadataItem.Key, metadataValue);
+                var metadataValueString = StachUtilities.ValueToString(metadataItem.Value.Value);
+                var metadataValue = metadataItem.Value.Value;
+
+                finalTable.Metadata.Add(metadataItem.Key, metadataValueString);
+
+                List<Google.Protobuf.WellKnownTypes.Value> valuesList = new List<Google.Protobuf.WellKnownTypes.Value>();
+                if (metadataValue.KindCase.ToString() == "ListValue")
```

Comment:
Is there any constant from where we can get the `ListValue` string? It is better to avoid hard coding.

Reply:
I'm unaware of any existing constants but I created new ones for `ListValue` and  `StringValue` here: https://github.com/factset/stach-extensions/pull/18/commits/b7f5b1c415bdc9896f952d3a3a202f0b62672d1c

Reply:
How about you dont do a ToString() and have a check if that equals "Value.KindOneofCase.ListValue".
You can refer similar matching used for a switch condition [here](https://github.com/factset/stach-extensions/blob/main/dotnet/StachExtensions/FactSet.Protobuf.Stach.Extensions/V2/StachUtilities.cs#L18)